spec/problem_one_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../problem_one'
+
+describe ProblemOne do
+  it 'should compute for a set of even numbers' do
+    p_one = ProblemOne.new
```

Comment:
Could you use `let`s? Check this: https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let